models/tridimensional/cas9_mutants.py

```diff
@@ -0,0 +1,800 @@
+    {'pam': 'NGC', 'backbone': 'R1335T+T1337R',
+     'mutations': [{'aa_idx': 1162, 'sec_structure': 'beta_strand_10', 'aa_init': 'E', 'aa_mut': 'D',
+                     'aa_group_init': 'acidic', 'aa_group_mut': 'acidic'},
+                    {'aa_idx': 1195, 'sec_structure': 'none', 'aa_init': 'I', 'aa_mut': 'M',
+                     'aa_group_init': 'hydrophobic', 'aa_group_mut': 'hydrophobic'},
+                    {'aa_idx': 1218, 'sec_structure': 'beta_strand_14', 'aa_init': 'G', 'aa_mut': 'R',
+                     'aa_group_init': 'special', 'aa_group_mut': 'basic'},
+                    {'aa_idx': 1309, 'sec_structure': 'helix_16', 'aa_init': 'I', 'aa_mut': 'V',
+                     'aa_group_init': 'hydrophobic', 'aa_group_mut': 'hydrophobic'},
+                    {'aa_idx': 1335, 'sec_structure': 'none', 'aa_init': 'R', 'aa_mut': 'T',
+                     'aa_group_init': 'basic', 'aa_group_mut': 'polar'},
+                    {'aa_idx': 1337, 'sec_structure': 'none', 'aa_init': 'T', 'aa_mut': 'R',
+                     'aa_group_init': 'polar', 'aa_group_mut': 'basic'}]},
+    {'pam': 'NGC', 'backbone': 'R1335T+T1337R',
+     'mutations': [{'aa_idx': 1135, 'sec_structure': 'none', 'aa_init': 'D', 'aa_mut': 'N',
+                     'aa_group_init': 'acidic', 'aa_group_mut': 'polar'},
+                    {'aa_idx': 1190, 'sec_structure': 'none', 'aa_init': 'V', 'aa_mut': 'I',
+                     'aa_group_init': 'hydrophobic', 'aa_group_mut': 'hydrophobic'},
+                    {'aa_idx': 1285, 'sec_structure': 'helix_15', 'aa_init': 'A', 'aa_mut': 'V',
+                     'aa_group_init': 'hydrophobic', 'aa_group_mut': 'hydrophobic'},
+                    {'aa_idx': 1335, 'sec_structure': 'none', 'aa_init': 'R', 'aa_mut': 'T',
+                     'aa_group_init': 'basic', 'aa_group_mut': 'polar'},
+                    {'aa_idx': 1337, 'sec_structure': 'none', 'aa_init': 'T', 'aa_mut': 'R',
+                     'aa_group_init': 'polar', 'aa_group_mut': 'basic'}]},
+    {'pam': 'NGC', 'backbone': 'R1335T+T1337R',
+     'mutations': [{'aa_idx': 1111, 'sec_structure': 'none', 'aa_init': 'L', 'aa_mut': 'H',
+                     'aa_group_init': 'hydrophobic', 'aa_group_mut': 'basic'},
+                    {'aa_idx': 1218, 'sec_structure': 'beta_strand_14', 'aa_init': 'G', 'aa_mut': 'R',
+                     'aa_group_init': 'special', 'aa_group_mut': 'basic'},
+                    {'aa_idx': 1297, 'sec_structure': 'turn_7', 'aa_init': 'H', 'aa_mut': 'D',
+                     'aa_group_init': 'basic', 'aa_group_mut': 'acidic'},
+                    {'aa_idx': 1335, 'sec_structure': 'none', 'aa_init': 'R', 'aa_mut': 'T',
+                     'aa_group_init': 'basic', 'aa_group_mut': 'polar'},
+                    {'aa_idx': 1337, 'sec_structure': 'none', 'aa_init': 'T', 'aa_mut': 'R',
+                     'aa_group_init': 'polar', 'aa_group_mut': 'basic'}]}
+]
```

Comment:
can we also move the list close to the end of the last dictionary entry? I think its clear enough that its a really long list lol